README.md

```diff
@@ -62,6 +61,107 @@ In a separate directory invoke the generator via
 yo ibm-service-enablement
 ```
 
+## Adding new services
+
+To add a new services you will need to follow these steps.
+
+Create a new composed generated by adding a service folder under generators with the prefix service- then the service id of your new service. To illustrate, if service called S3 needs to be added the folder would be called service-s3.
```

Comment:
This sentence is confusing. 

What do you mean by "Create a new composed generated .."?

"generators" --> "`generators/`"
" prefix service-" --> "prefix `service-`"
"service-s3" --> "`service-s3`"